sherlock/internal/api/sherlock/chart_version_v3_list.go

```diff
@@ -0,0 +1,62 @@
+package sherlock
+
+import (
+	"fmt"
+	"github.com/broadinstitute/sherlock/go-shared/pkg/utils"
+	"github.com/broadinstitute/sherlock/sherlock/internal/authentication"
+	"github.com/broadinstitute/sherlock/sherlock/internal/errors"
+	"github.com/broadinstitute/sherlock/sherlock/internal/models"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+// chartVersionsV3List godoc
+//
+//	@summary		List ChartVersions matching a filter
+//	@description	List ChartVersions matching a filter.
+//	@tags			ChartVersions
+//	@produce		json
+//	@param			filter					query		ChartVersionV3	false	"Filter the returned ChartVersions"
+//	@param			limit					query		int		false	"Control how many ChartVersions are returned (default 100)"
+//	@param			offset					query		int		false	"Control the offset for the returned ChartVersions (default 0)"
+//	@success		200						{array}		ChartVersionV3
+//	@failure		400,403,404,407,409,500	{object}	errors.ErrorResponse
+//	@router			/api/chart-versions/v3 [get]
+func chartVersionsV3List(ctx *gin.Context) {
+	db, err := authentication.MustUseDB(ctx)
+	if err != nil {
+		return
+	}
+	var filter ChartVersionV3
+	if err = ctx.ShouldBindQuery(&filter); err != nil {
+		errors.AbortRequest(ctx, err)
+		return
+	}
+	modelFilter, err := filter.toModel(db, true)
+	if err != nil {
+		errors.AbortRequest(ctx, fmt.Errorf("(%s) %v", errors.BadRequest, err))
+		return
+	}
+
+	limit, err := utils.ParseInt(ctx.DefaultQuery("limit", "100"))
+	if err != nil {
+		errors.AbortRequest(ctx, fmt.Errorf("(%s) %v", errors.BadRequest, err))
+		return
+	}
+	offset, err := utils.ParseInt(ctx.DefaultQuery("offset", "0"))
+	if err != nil {
+		errors.AbortRequest(ctx, fmt.Errorf("(%s) %v", errors.BadRequest, err))
+		return
+	}
+	var results []models.ChartVersion
+	if err = db.
+		Where(&modelFilter).
+		Limit(limit).
+		Offset(offset).
+		Order("id asc").
```

Comment:
```suggestion
		Order("created_at desc").
```

I think we actually probably want to change this for https://github.com/broadinstitute/sherlock/blob/main/sherlock/internal/api/sherlock/app_version_v3_list.go#L56 too. The IDs are a Sherlock-internal thing so they're pretty meaningless to sort by, and I think Beehive does stuff like `/api/v2/chart-versions?limit=10` to get the ten most recent app versions, that kind of thing.